src/main/java/com/faforever/client/util/TimeService.java

```diff
@@ -80,6 +91,38 @@ private Locale getCurrentDateLocale() {
 
   }
 
+  private String getDatePatternForDayMonthYear(FormatStyle style) {
+    switch (style) {
+      case SHORT -> {
+        return "d/M/yy";
+      }
+      case MEDIUM -> {
+        return "d MMM, yyyy";
+      }
+      default -> {
+        // for other styles extend this
+        return "d MMMM, yyyy";
+      }
```

Comment:
Why is default used instead of an exhaustive switch? 

Also use a switch expression instead of statement.

Reply:
Well, because there are Long and Full FormatStyle that are not implemented -> in this case, default will be able to deal Both as Long style. From the code I could see usage of short and medium styles.

Additionally, I dont think it is worth it to throw exception and break application, if somebody passed weird style. Though, if you want it, I don't mind to change it. Just let me know.

Reply:
I would not throw an exception but would just make explicit the long and full instead of using default.